Unify identify and user property docs (#6105)
PostHog/posthog.com: contents/docs/libraries/js/index.mdx

<JSCapture />

### Setting user properties

> To set [user properties](/docs/data/user-properties), include the properties you'd like to set when capturing an event:

Reviewer: ```suggestion
To set [user properties](/docs/data/user-properties), include them when capturing an event:
```

> ```javascript
> posthog.capture(
>     'event_name', 
>     { 
>         $set: { name: 'Max Hedgehog'  },
>         $set_once: { initial_url: '/blog' },
>     }
> )

Reviewer: ```suggestion
```javascript
posthog.capture(
  'event_name', 
  { 
    $set: { name: 'Max Hedgehog'  },
    $set_once: { initial_url: '/blog' },
  }
)
```

```

For more details on the difference between `$set` and `$set_once`, see our [user properties docs](/docs/data/user-properties#what-is-the-difference-between-set-and-set_once).

### Identifying users

> > We strongly recommend reading our docs on [identifying users](/docs/integrate/identifying-users) to better understand how to correctly use this method.

Reviewer: Update link here?

import JSIdentify from './\_snippets/identify.mdx'

<JSIdentify />

### Alias

> > We strongly recommend reading our docs on [alias](/docs/integrate/identifying-users#alias-assigning-multiple-distinct-ids-to-the-same-user) to better understand how to correctly use this method.

Reviewer: Update link here?

> Sometimes, you may want to assign multiple distinct IDs to a single user. This is helpful in scenarios where your primary distinct ID may be inaccessible. For example, if a distinct ID which is typically used on the frontend is not available in certain parts of your backend code. In this case, you can use `alias` to assign another distinct ID to the same user:

Reviewer: ```suggestion
Sometimes, you want to assign multiple distinct IDs to a single user. For example, if a distinct ID which is typically used on the frontend is not available in certain parts of your backend code, you can use `alias` to connect the frontend distinct ID to one accessible on the backend.
```

```javascript
posthog.alias('alias_id', 'distinct_id');
```

### Reset after logout

> If a user logs out, you should call `reset` to unlink any future events made on that device with that user. 

Reviewer: Should we use the identify reset snippet here?

**Automatic overrides**

> Whenever you call `posthog.identify` with person properties, we automatically add these properties to flag evaluation calls. The same is true for when you call `posthog.group()`.

Reviewer: It isn't obvious why this matters. Could you add one more sentence about that?
